2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -296,6 +297,7 @@ kafka_validate() {
     elif ! is_boolean_yes "$ALLOW_PLAINTEXT_LISTENER"; then
          print_validation_error "The KAFKA_ZOOKEEPER_PROTOCOL environment variable does not configure a secure protocol. Set the environment variable ALLOW_PLAINTEXT_LISTENER=yes to allow the container to be started with a plaintext listener. This is only recommended for development."
     fi
+    check_multi_value "KAFKA_CFG_SSL_CLIENT_AUTH" "none requested required"
```

Comment:
You're treating the 3 words as a single string:
```suggestion
    check_multi_value "KAFKA_CFG_SSL_CLIENT_AUTH" "none" "requested" "required"
```